chrome/locale/en-US/zotero/preferences.dtd

```diff
@@ -60,9 +60,16 @@
 <!ENTITY zotero.preferences.sync.fileSyncing.url					"URL:">
 <!ENTITY zotero.preferences.sync.fileSyncing.myLibrary				"Sync attachment files in My Library using">
 <!ENTITY zotero.preferences.sync.fileSyncing.groups				"Sync attachment files in group libraries using Zotero storage">
+<!ENTITY zotero.preferences.sync.fileSyncing.groups.global			"Global Settings">
+<!ENTITY zotero.preferences.sync.fileSyncing.groups.custom			"Use Custom Settings">
+<!ENTITY zotero.preferences.sync.fileSyncing.groups.sync			"Sync attachment files">
+<!ENTITY zotero.preferences.sync.fileSyncing.groups.customize		"Customize...">
+<!ENTITY zotero.preferences.sync.fileSyncing.groups.revert			"Revert All Groups to Global Settings">
 <!ENTITY zotero.preferences.sync.fileSyncing.download				"Download files">
 <!ENTITY zotero.preferences.sync.fileSyncing.download.atSyncTime	"at sync time">
 <!ENTITY zotero.preferences.sync.fileSyncing.download.onDemand		"as needed">
+<!ENTITY zotero.preferences.sync.fileSyncing.remove				    "Remove downloaded files if not opened for">
+<!ENTITY zotero.preferences.sync.fileSyncing.remove.suffix			"day(s)">
```

Comment:
I prefer to avoid this, which is a little clumsy and also possibly not easily localizable. That's why it might be better to put a few time and unit options in the drop-down ("1 day", "1 week", "1 month", "3 months"), even if it's less customizable. That also has the advantage of letting you quickly configure everything with the mouse, instead of having to keep switching between the mouse and the keyboard across many groups.

Assuming we still stored the pref as days, someone could still customize in the Config Editor, and then I guess we'd want to display a "[x] days" value in the list, at the top — a few minutes' extra work, but why not? (That should be localized in zotero.properties as `%1$S day;%1$S days`, with the number given a second time as the third parameter to `getString()` — see `findPDF.pdfsAdded` in attachments.js for an example. Other languages with different numbers of plural forms will localize differently.)

(For emptying the trash in the General pane, we do use "days", since that defaults to 30 and you probably don't want it to be 1 anyway, but for configuring multiple times here the drop-down seems a little easier to me.)

Reply:
This makes sense. I was trying to be clever and minimizing the number of translation, I'm not sure why. Thanks for the detail.